include/TRFFitter.h

```diff
@@ -3,8 +3,8 @@
 
 #include <vector>
 
-#include "TNamed.h"
-#include "Rtypes.h"
+#include <TNamed.h>
+#include <Rtypes.h>
```

Comment:
not entirely.  we do something simular with the addback window in the
tgrsiruninfo i believe, where we pass an acceptable time window to which
events get built together, but for the silicon classes this has mostly been
hard coded(see csm and sharc) into the build events so far.  The difference
between a si and a ge event is that for a si event, the front back
coincidence should be at exactly the same time plus or minus any clock
jitter while for a ge event, the time is rise time dependent and hence open
to more interpretation.

I would make a delta-t plot for the front back fragments from a fragment
tree, I've got dollars to dounts that the coincidence window size you want
for an s3 is fairly obvious and additionally will be fairly constant across
experiments,  On a longer term, there is nothing preventing us from adding
a spice front-back coincidence window to the runifno, if set it would be
used, otherwise we could default to some "reasonable value"

On Wed, Nov 4, 2015 at 9:10 PM, jsmallcombe <notifications@github.com>
wrote:

> In include/TRFFitter.h
> <https://github.com/GRIFFINCollaboration/GRSISort/pull/584#discussion_r43969038>
> :
>
> > @@ -3,8 +3,8 @@
> >
> >  #include <vector>
> >
> > -#include "TNamed.h"
> > -#include "Rtypes.h"
> > +#include <TNamed.h>
> > +#include <Rtypes.h>
>
> Happy to oblige that.
> Do we have a list of such style things?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GRIFFINCollaboration/GRSISort/pull/584/files#r43969038>
> .
>